### PR TITLE
:sparkles: update ReturnReason schemas to include possible description object

### PR DESCRIPTION
**Pull request by yoerriwalstra**
**Related issue**
https://myparcelcombv.atlassian.net/browse/MP-6471

### CODE REVIEW

File: specification/paths/Returns-v1-reasons.json [hunk 1]
```diff
@@ -20,6 +20,15 @@
...
         "$ref": "#/components/parameters/query-filter-tags"
+      },
+      {
+        "in": "header",
+        "name": "Accept-Language",
+        "schema": {
+          "type": "string"
+        },
+        "description": "Description of your optional header",
+        "required": false
```
Comment (M4tini) [inline specification/paths/Returns-v1-reasons.json:31]:
```suggestion
        "description": "Determines the language in which the ``description`` is returned,"
```
and maybe something about "ISO 3166-1 alpha-2 compliant language tag" like you mentioned below

File: specification/schemas/return-reasons/ReturnReason.json [hunk 2]
```diff
@@ -15,8 +15,17 @@
...
-              "example": "The item isn't what I expected it to be."
+              "oneOf": [
+                {
+                  "$ref": "#/components/schemas/ReturnReasonManagementDescription"
+                },
+                {
+                  "type": "string",
+                  "description": "A localized return reason description",
+                  "example": "The item isn't what I expected it to be."
+                }
```
Comment (M4tini) [inline specification/schemas/return-reasons/ReturnReason.json:26]: I would swap these `oneOf` elements, so the JSON example shows the "string" version by default. This is more clear for regular people using the endpoint (the management version is mainly for us).

**Review by M4tini — CHANGES_REQUESTED** [review]
I think the following endpoints should implement the header and default to `en-GB` as well.
- `/returns/v1/reasons/{return_reason_id}`
- `/returns/v1/shops/{shop_id}/reasons`

File: specification/paths/Returns-v1-reasons.json (hunk 1 above)
Comment (yoerriwalstra) [reply specification/paths/Returns-v1-reasons.json]: done

File: specification/schemas/return-reasons/ReturnReason.json (hunk 2 above)
Comment (yoerriwalstra) [reply specification/schemas/return-reasons/ReturnReason.json]: done

**Review by M4tini — CHANGES_REQUESTED** [review]
Let's also add the header to `/returns/v1/reasons/{return_reason_id}` and move the header to a reusable `#/components/parameters/header-accept-language`

**Comment by yoerriwalstra** [comment]
> Let's also add the header to /returns/v1/reasons/{return_reason_id} and move the header to a reusable #/components/parameters/header-accept-language

Done

**Review by M4tini — APPROVED** [review]

**Pull request merged**